sysbuild/CMakeLists.txt

```diff
@@ -105,6 +109,10 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_post_cmake)
     include_provision_hex()
   endif()
 
+  if(DEFINED FP_MODEL_ID AND DEFINED FP_ANTI_SPOOFING_KEY)
+    include_fast_pair_hex()
```

Comment:
Tested locally. This function seems not called at all (the FP_MODEL_ID and FP_ANTI_SPOOFING_KEY are not defined here).

Reply:
Also tested locally with this PR rebased on top of https://github.com/nrfconnect/sdk-nrf/pull/12918. This function is called if `FP_MODEL_ID` and `FP_ANTI_SPOOFING_KEY` are provided during build e.g. `-DFP_MODEL_ID=0x8E717D -DFP_ANTI_SPOOFING_KEY=dZxFzP7X9CcfLPC0apyRkmgsh3n2EbWo9NFNXfVuxAM=`. However, `samples/bluetooth/fast_pair/input_device` provides possibility of building Fast Pair with default keys. See samples/bluetooth/fast_pair/input_device/CMakeLists.txt`:
```
if(NOT DEFINED FP_MODEL_ID AND NOT DEFINED FP_ANTI_SPOOFING_KEY)
  message(WARNING "
  -------------------------------------------------------
  --- WARNING: Using demo Fast Pair Model ID and Fast ---
  --- Pair Anti Spoofing Key, it should not be used   ---
  --- for production.                                 ---
  -------------------------------------------------------
  \n"
  )
  set(FP_MODEL_ID "0x2A410B")
  set(FP_ANTI_SPOOFING_KEY "Unoh+nycK/ZJ7k3dHsdcNpiP1SfOy0P/Lx5XixyYois=")
endif()
```
In this case, the `include_fast_pair_hex()` is not called. Seems like the `set(FP_MODEL_ID "0x2A410B")` is somehow overlooked even though I can see that this part is being executed, because I can see Cmake warning.

Reply:
Ah so you're building the same without any options on the command line, that would do it, will submit a fix

Reply:
Fixed

Reply:
It works, but the warning message: 
'''
 message(WARNING "
  -------------------------------------------------------
  --- WARNING: Using demo Fast Pair Model ID and Fast ---
  --- Pair Anti Spoofing Key, it should not be used   ---
  --- for production.                                 ---
  -------------------------------------------------------
  \n"
'''
is printed twice. Maybe it could be put inside `if(NOT SYSBUILD)`?